dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -16,6 +20,24 @@ namespace AppNotificationBuilder
     using namespace winrt::Microsoft::Windows::AppNotifications::Builder;
 }
 
+inline std::unordered_map<wchar_t, std::wstring> GetXmlEscapeEncodings()
+{
+    static std::unordered_map<wchar_t, std::wstring> encodings = { { L'&', L"&amp;"}, { L'\"', L"&quot;"}, {L'<', L"&lt;"}, {L'>', L"&gt;"}, {L'\'', L"&apos;"}};
+    return encodings;
+}
+
+inline std::unordered_map<wchar_t, std::wstring> GetPercentEncodings()
+{
+    static std::unordered_map<wchar_t, std::wstring> encodings = {{ L'%', L"%25"}, {L';', L"%3B"}, {L'=', L"%3D"} };
+    return encodings;
+}
+
+inline std::unordered_map<std::wstring, wchar_t> GetPercentEncodingsReverse()
+{
+    static std::unordered_map<std::wstring, wchar_t> encodings = { { L"%25", L'%' }, {L"%3B", L';' }, { L"%3D", L'=' } };
```

Comment:
Same comment as above here. Why return a list when you can just return a char and the API takes in an encodedString